bin/run_lint

```diff
@@ -160,12 +160,16 @@ lint rest_api/sawtooth_rest_api "$SINCE" || retval=1
 
 PYTHONPATH=$top_dir/families/settings
 PYTHONPATH=$top_dir/families/identity
+PYTHONPATH=$top_dir/families/block_info
+PYTHONPATH=$top_dir/families/battleship
 PYTHONPATH=$PYTHONPATH:$top_dir/signing
 PYTHONPATH=$PYTHONPATH:$top_dir/families/supplychain/python
 PYTHONPATH=$PYTHONPATH:$top_dir/sdk/python
 export PYTHONPATH
 lint families/settings "$SINCE" || retval=1
 lint families/identity "$SINCE" || retval=1
+lint families/block_info "$SINCE" || retval=1
+lint families/battleship "$SINCE" || retval=1
```

Comment:
Copy the pattern for block_info below, don't add this to the list of families.

Reply:
Fixed